proofs/cbmc/poly_uniform/poly_uniform_harness.c

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2025 The mldsa-native project authors
+// SPDX-License-Identifier: Apache-2.0
+
+#include "fips202/fips202.h"
+#include "poly.h"
+
+void poly_uniform(poly *a, const uint8_t seed[MLDSA_SEEDBYTES], uint16_t nonce);
```

Comment:
`poly_uniform` is declared in `poly.h`, so no need to re-declare it here.

Reply:
fixed!